src/symbols_macos.cpp

```diff
@@ -14,6 +14,19 @@
 #include "symbols.h"
 #include "log.h"
 
+UnloadProtection::UnloadProtection(CodeCache *cc) {
+    _protected_cc = cc;
```

Comment:
`_protected_cc` is no longer used.

Reply:
Removed all occurrences, thanks: 2abe2aa238474e648137c8eca8fcfcf1499fa430